test/unit/app/utils/test_kenya_state_machine.py

```diff
@@ -183,6 +186,7 @@ def test_opt_out_of_marketplace(mocker, test_client, init_database):
     state_machine.send_sms.assert_called_with("opt_out_of_market_place_sms")
 
 
+#TODO: also test that users pin becomes the correct thing after
```

Comment:
Does this TODO refer to a test for the pin changing post `complete_initial_pin_change` and `complete_pin_change` ?